css-positioning/css-positioning_absolute-no-advanced-requirements/css/styles.css

```diff
@@ -0,0 +1,84 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: system-ui;
+}
+
+main {
+  width: 500px;
+  height: 600px;
+  border: 1px solid black;
+  margin: 5em auto;
+  position: relative;
+}
+
+.box {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 80px;
+  width: 80px;
+  text-align: center;
+}
+
+/**
+ * Look at the selectors defined below.
+ * Position each element, so that the placement matches the name of the selector.
+ * Adjust the rule-sets for each selector to satisfy the name.
+ *
+ * @example
+ * .box--top-left -> This element should be be placed in the top, left corner.
+*/
+
+.box--top-left {
+  position: absolute;
+  background-color: #aed6f1;
+}
+
+.box--top-center {
+  background-color: #85c1e9;
+  position: absolute;
+  top: 0;
+  left: 210px;
+}
+
+.box--top-right {
+  position: absolute;
+  top: 0;
+  right: 0;
+
+  background-color: #5dade2;
+}
+
+.box--center {
+  background-color: #3498db;
+
+  position: absolute;
+  top: 260px;
+  left: 210px;
+}
+
+.box--bottom-left {
+  background-color: #2874a6;
+  position: absolute;
+  top: 520px;
```

Comment:
Nothing wrong, but it might be more convenient to change from "top: 520px" to "bottom: 0"